bootstrap/postgres/setup_test.go

```diff
@@ -9,8 +9,8 @@ import (
 	"testing"
 
 	"github.com/jmoiron/sqlx"
-	bootstrapRepo "github.com/mainflux/mainflux/bootstrap/postgres"
-	pgClient "github.com/mainflux/mainflux/internal/clients/postgres"
+	"github.com/mainflux/mainflux/bootstrap/postgres"
+	pgclient "github.com/mainflux/mainflux/internal/clients/postgres"
 	"github.com/mainflux/mainflux/logger"
 
 	dockertest "github.com/ory/dockertest/v3"
```

Comment:
```go
dockertest "github.com/ory/dockertest/v3"
```
This is not needed as the package is `dockertest`. This applies to all occurrences